src/framework/types/serializable/resolvable.ts

```diff
@@ -23,6 +23,11 @@ export type ResolvableEffectWithoutSource = {
     ;
   player: PlayerParam;
   amount: ConstantParam<number>;
+} | {
+  type: "SUMMON_TOKEN";
```

Comment:
Does our effect chain system currently have a way to capture the ID of the created token? I can see cases where we might need to have "summon a token, then do other stuff to it". Maybe that could be worked into onEntersPlay for the token itself though?

Reply:
Considering that multiple tokens can be summoned in a single action, I guess it would have to be an array of tokens. Still, that's a good idea. Terras Q comes to mind as a card that would have some unfortunate edge cases if we had to look up tokens after the fact (e.g. if there are multiple Terras Q's in play in a single game).